utils/build.ps1

```diff
@@ -1709,7 +1709,7 @@ function Build-Compilers() {
       })
   }
 
-  Invoke-Program "$(Get-PythonExecutable)" -c "import plistlib; print(str(plistlib.dumps({ 'Identifier': '${ToolchainIdentifier}' }), encoding='utf-8'))" `
+  Invoke-Program "$(Get-PythonExecutable)" -c "import plistlib; print(str(plistlib.dumps({ 'Identifier': '${ToolchainIdentifier}', 'FallbackLibrarySearchPaths': ['usr/bin'], 'Version': '${ProductVersion}', 'DefaultBuildSettings': { 'TOOLCHAIN_VERSION': '${ProductVersion}' } }), encoding='utf-8'))" `
```

Comment:
I'm not thrilled about the idea of further proliferating build settings outside of swift-build itself. Could we handle `TOOLCHAIN_VERSION` differently, such as making it a top-level "ToolchainVersion" plist key?

Reply:
I removed it for now because it's easy enough to just derive the contents from the Version field

Reply:
Oh, right, there was even an existing `Version` field -- thanks!